pynq/lib/audio.py

```diff
@@ -342,20 +344,22 @@ def __init__(self, description):
         self._ffi.cdef("""void select_mic(int iic_index);""")
         self._ffi.cdef("""void deselect(int iic_index);""")
         self._ffi.cdef("""void bypass(unsigned int audio_mmap_size,
-                          unsigned int nsamples, 
+                          unsigned int nsamples, unsigned int volume,
                           int uio_index, int iic_index) ;""")
         self._ffi.cdef("""void record(unsigned int audio_mmap_size,
                           unsigned int * BufAddr, unsigned int nsamples,
                           int uio_index, int iic_index);""")
         self._ffi.cdef("""void play(unsigned int audio_mmap_size,
                           unsigned int * BufAddr, unsigned int nsamples,
+                          unsigned int volume,
                           int uio_index, int iic_index);""")
 
         self.buffer = numpy.zeros(0).astype(numpy.int32)
         self.sample_rate = None
         self.sample_len = len(self.buffer)
         self.iic_index = None
         self.uio_index = None
+        self.volume = 57
         self.configure()
```

Comment:
Please remove this line as it conflicts with changes made in https://github.com/Xilinx/PYNQ/pull/1029. We are getting rid of this call here.

Reply:
I think @giunatale means to remove the line `self.configure()`?

Reply:
Indeed. Sorry I pointed to the wrong line.

Reply:
Should be resolved now.

Reply:
I am not sure, others might comment and clarify in case I am wrong. Since you merged the commit coming from https://github.com/Xilinx/PYNQ/pull/1029, it will probably give a merge-conflict when trying to merge both PRs. The best option would be that you revert the commit altoghether and just remove the line in question from audio.py in a new commit. Please, also force-push these changes as it will keep history clean.

Reply:
As nobody has replied to your question, I did as you requested. But just to clarify. I think I did the right thing. The conflicting changes between PRs should be handled by the maintainers of a repository. If two PRs have conflicts, there should be a merge commit. Otherwise, there will be commits reintroducing the same lines over and over again. Just my opinion.